classification/metrics/compute_vessel_metrics.py

```diff
@@ -694,6 +694,7 @@ def load_inferred(inference_table, label_table, extractors):
     JOIN
    `{}*` inference_table
     ON (cast(label_table.id as string) = inference_table.ssvid)
+    where split = "Test"
```

Comment:
@bitsofbits I think this is supposed to be removed, right?

Reply:
@enriquetuya , I believe it should be as it is (we are only computing metrics against the Test split), but I'm going to take a look now to confirm.

Reply:
@enriquetuya : This is OK. This is for computing the vessels characterization metrics after a run, so we only load the vessels in the test split since those are what we want to compute metrics on.